samples/SuppressAlerts.ps1

```diff
@@ -0,0 +1,55 @@
+param($TargetHostName, $ZabbixAPIUrl)
+
+Import-Module PSZabbix
+
+Write-Host "Using Zabbix API endpoint: $ZabbixAPIUrl"
+
+#
+# Default Zabbix security protocol was not supported by newer PowerShell in my test bed.
+# I had to tell PowerShell to use an older protocol
+#
+$defaultSecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol
```

Comment:
Depending on the way Zabbix site is hosted (it will be very often behind a reverse proxy) it could actually change. So I would simply add a comment in the documentation "here is how to use TLS1.2".

Reply:
Thanks - I will make these changes. Aside, the module is at the beginnings of use on the team and seems well received.